data-prepper-plugins/kafka-plugins/src/main/java/org/opensearch/dataprepper/plugins/kafka/consumer/KafkaSourceCustomConsumer.java

```diff
@@ -176,12 +170,27 @@ public <T> void consumeRecords() throws Exception {
                 }
             }
         } catch (AuthenticationException e) {
-            LOG.warn("Authentication Error while doing poll(). Will retry after 10 seconds", e);
+            LOG.warn("Access Denied while doing poll(). Will retry after 10 seconds", e);
```

Comment:
The [Apache documentation](https://kafka.apache.org/20/javadoc/org/apache/kafka/common/errors/AuthenticationException.html) indicates that this is an authentication error and not necessarily related to access controls.